### PR TITLE
Replacing Conductor for Containers

### PR DESCRIPTION
**Pull request by kealexan**
Replacing CfC with IBM Cloud private.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4311)

### CODE REVIEW

**Comment by kealexan** [comment]
Testing CNCF-CLA status

**Comment by kealexan** [comment]
^^

File: docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ a Kubernetes cluster from scratch.
 
 * [Ubuntu on LXD](/docs/getting-started-guides/ubuntu/local) supports a nine-instance deployment on localhost.
 
-* [IBM Spectrum Conductor for Containers](https://www.ibm.com/support/knowledgecenter/SS8TQM/product_welcome_containers.html) can use VirtualBox on your machine to deploy Kubernetes to one or more VMs. Scales to full multi-node cluster. Free distribution.
+* [IBM Cloud local-ce (Community Edition)](https://www.ibm.com/support/knowledgecenter/en/SSBS6K/product_welcome_cloud_private.html) can use VirtualBox on your machine to deploy Kubernetes to one or more VMs for dev and test scenarios. Scales to full multi-node cluster. Free version of the enterprise solution.
```
Comment (gyliu513) [inline docs/setup/pick-right-solution.md]: @kealexan Should not this be `IBM Cloud private-ce`?

Comment (kealexan) [inline docs/setup/pick-right-solution.md]: Yes, thank you! I opened a new PR: https://github.com/kubernetes/kubernetes.github.io/pull/4347

**Pull request merged**